tools/mo/openvino/tools/mo/main.py

```diff
@@ -447,6 +479,12 @@ def main(cli_parser: argparse.ArgumentParser, fem: FrontEndManager, framework: s
             argv.framework = framework
         argv.feManager = fem
 
+        # Set which frontend to use by default, values should be 'new' or 'legacy'
+        argv.frontend_defaults = {
+            'onnx': 'legacy',
+            'tf': 'legacy'
+        }
```

Comment:
please revert this change and move to the previous place

Reply:
done, but I've moved it to a separate function to allow mocking in UTs